config/routes.rb

```diff
@@ -10,11 +10,15 @@
   }
   get "up" => "rails/health#show", as: :rails_health_check
 
-  resources :experts, only: [:index, :new, :create, :show ] do
-    member do
-      put :toggle_remove
+  namespace :api do
+    namespace :v1 do
+      resources :experts, only: [:index, :new, :create, :show ] do
+        member do
+          put :toggle_remove
+        end
+        resources :specializations, only: [:index,:new, :show]
+      end
```

Comment:
I think here you can collect all routes inside one with out duplicating namespace :API do
    namespace: v1 do